pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -24,6 +24,8 @@ import (
 const resolvConf = "/etc/resolv.conf"
 const defaultDNSPort = 53
 const lameDuckDuration = 20 * time.Second
+const cacheDefaultMaxPositiveTTL = 900
```

Comment:
As we don't use `* time.Second` but `uint32`, it is not obvious if these are in seconds. You can use `InSeconds` in the constant name to make it more obvious.

Reply:
Good call! I added a note about why they're integers instead of `time.Duration` and refactored the names to include `Seconds`.